requirements.txt

```
@@ -1,32 +1,53 @@
 beautifulsoup4==4.6.3
+Brotli==1.0.9
 certifi==2018.11.29
+cffi==1.15.1
 chardet==3.0.4
+charset-normalizer==3.0.1
+click==8.1.3
+colorama==0.4.6
+csscompressor==0.9.5
 cssselect2==0.2.1
+defusedxml==0.5.0
+fonttools==4.38.0
+ghp-import==2.1.0
 gitdb==4.0.5
 GitPython==3.1.30
-defusedxml==0.5.0
+html5lib==1.1
+htmlmin==0.1.12
 idna==2.8
 Jinja2==3.0.3
+jsmin==3.0.1
 Markdown==3.2.1
 markdown-captions==2
-mkdocs==1.2.3
-pygments==2.11.2
-mkdocs-material==7.3.4
-mkdocs-minify-plugin==0.2.1
+MarkupSafe==2.1.1
+mergedeep==1.3.4
+mkdocs==1.4.2
+mkdocs-material @ https://github.com/Philip-Larsen-Donnelly/mkdocs-material/archive/refs/tags/d2mm-9.0.8.3.zip
```

Incorporate change to boost titles higher than tags
```suggestion
mkdocs-material @ https://github.com/Philip-Larsen-Donnelly/mkdocs-material/archive/refs/tags/d2mm-9.0.8.4.zip
```